package.json

```diff
@@ -126,8 +149,11 @@
     "jshint-stylish": "2.2.1",
     "json-loader": "0.5.7",
     "lodash": "4.17.11",
+    "mapbox-gl": "0.53.1",
```

Comment:
Just noting that we'll need to clean at least this dependency out from Calypso root `package.json`:

https://github.com/Automattic/wp-calypso/blob/e4862897aec6ccd7e3ef7120df19f20df216a07a/package.json#L143

I expected to find it in Jetpack blocks' `package.json` instead so that's why I'm flagging it here: https://github.com/Automattic/wp-calypso/blob/612b387384a9a85b7c45238650af7c0ef2e724c3/packages/jetpack-blocks/package.json

Reply:
Should pull that into https://github.com/Automattic/wp-calypso/pull/31759

I believe also the `markdown-it` dep 👍